contracts/.env.example

```diff
@@ -8,12 +8,17 @@ BRIGHTID_CONTEXT=clr.fund
 # BrightId node addr that signs verifications. Node One uses this one
 BRIGHTID_VERIFIER_ADDR=0xb1d71F62bEe34E9Fc349234C201090c33BCdF6DB
 
-RINKEBY_JSONRPC_HTTP_URL=https://eth-rinkeby.alchemyapi.io/v2/ADD_API_KEY
-XDAI_JSONRPC_HTTP_URL=https://rpc.xdaichain.com
-RINKARBY_JSONRPC_HTTP_URL=
+# JSON-RPC endpoint to the selected network
+JSONRPC_HTTP_URL=https://eth-rinkeby.alchemyapi.io/v2/ADD_API_KEY
 
+# One of the two options
 WALLET_MNEMONIC=
 WALLET_PRIVATE_KEY=
 
 # Token address for funding round .setToken
 NATIVE_TOKEN_ADDRESS=
+
+# Required to use the tally and finalize scripts
+ROUND_ADDRESS=
+COORDINATOR_PK=
+COORDINATOR_ETH_PK=
```

Comment:
Note setting these env. variables in the finalize script docs

Reply:
To be super clear, which docs exactly are you talking about? the README.md or to add comments on top of the function itself, in the `scripts/finalize.ts` file, or something else?

Reply:
Valid question, I meant in the coordinator docs: https://github.com/ethereum/clrfund/blob/develop/docs/coordinator.md